Games/Star Trek Online/Steam/script.js

```diff
@@ -0,0 +1,12 @@
+ 
+include(["Wine", "QuickScript", "SteamScript"]);
+
+new SteamScript()
+    .name("Star Trek Online")
+    .editor("Cryptic Studios")
+    .author("ImperatorS79")
+    .appId(9900)
+    .wineVersion("2.12")
+    .wineDistribution("staging")
+    
+.go();
```

Comment:
Just to be consistent with the other scripts: could you remove the first empty line (also in script.json) and indent the `.go()`?